library/oraclelinux

```diff
@@ -4,10 +4,10 @@ GitCommit: 5c8a1c296acd6e90487cd261d16cf85fd6bcb73f
 GitFetch: refs/heads/master
 # https://github.com/oracle/container-images/tree/dist-amd64
 amd64-GitFetch: refs/heads/dist-amd64
-amd64-GitCommit: 294d682bff4740ce534309f766bf0f6736bf7be6
+amd64-GitCommit: d66749be81ce303cf6c5cb09a36f60aa0d63b671
 # https://github.com/oracle/container-images/tree/dist-arm64v8
 arm64v8-GitFetch: refs/heads/dist-arm64v8
-arm64v8-GitCommit: 1ca940d0aaa06cccf3bb1c6917adda069c8a11ad
+arm64v8-GitCommit: 3ed3cec468a85a641d53b3654058e715c3ee905b
 
```

Comment:
😕 I don't see any sections pointing to OL `10` or `10-slim` images. Perhaps something like this needs to be added?
```
Tags: 10
Architectures: amd64, arm64v8
Directory: 10